Azure/sonic-buildimage
# [202012] Split kvmtest t0 job into two jobs and run in parallel

Signed-off-by: Saikrishna Arcot <sarcot@microsoft.com>
(cherry picked from commit 44028723ef76385958af1555ad7f7150f53f44ee)



#### Why I did it

202012 branch of sonic-mgmt has the KVM T0 tests split up into 2 parts. However, the 202012 branch of sonic-buildimage doesn't know about these two parts, and so only half of the tests are being run.

#### How I did it

Cherry-pick the commit that added support for running these 2 parts from master branch to 202012 branch.

#### How to verify it

Verify that part 1 and part 2 both run, and that the time taken to run the tests is longer than the current time.

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog


#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
